packages/cli/test/dev/fixtures/go/api/[segment].go

```diff
@@ -1,10 +0,0 @@
-package handler
-
-import (
-	"fmt"
-	"net/http"
-)
-
-func Handler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprintf(w, "Req Path: %s", r.URL.Path)
-}
```

Comment:
The correct behavior doesn't currently work. It will be restored in a follow-up PR.

Reply:
Lets comment it out instead? Seems pretty critical to this test since the fixture is meant to test path segments like this

Reply:
I can't comment the contents because the file name is what causes the problem. I could rename the file, but I don't think it's that much different. I'm going to restore this file as soon as possible.